web_console/dynamic-plug-ins.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: ASSEMBLY
+[id="adding-dynamic-plugin-to-console"]
+= Adding a dynamic plug-in to the {product-title} web console
+include::modules/common-attributes.adoc[]
+:context: dynamic-plugins
+
+toc::[]
+
+You can create and deploy a dynamic plug-in on your cluster that is loaded at run-time.
+
+:FeatureName: Creating a dynamic plug-in
+include::snippets/technology-preview.adoc[leveloffset=+1]
+//
+
+include::modules/about-dynamic-plug-ins.adoc[leveloffset=+1]
+
+include::modules/dynamic-plug-ins-get-started.adoc[leveloffset=+1]
+
+include::modules/running-your-dynamic-plug-in.adoc[leveloffset=+1]
+
+include::modules/adding-new-extension-dynamic-plug-in.adoc[leveloffset=+1]
+
+include::modules/build-image-docker.adoc[leveloffset=+1]
+
+include::modules/deployment-plug-in-cluster.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="dynamic-plugin-additional-resources"]
+== Additional resources
+
+* The console customization plug-in can provide some guidance to building a dynamic plug-in, refer to link:https://github.com/spadgett/console-customization-plugin[Console customization plug-in].
```

Comment:
```suggestion
* link:https://github.com/spadgett/console-customization-plugin[Console customization plug-in].
```
I think this is the new format required for additional resources in Jupiter. I can ask in the team chat for verification.

Reply:
Made the change. I will verify this.